### PR TITLE
Add inszenium

### PR DESCRIPTION
**Pull request by katgirl**

### CODE REVIEW

**Comment by xchs** [comment]
You might want to optimize the logo with [SVGO](https://jakearchibald.github.io/svgomg/)?

File: meta/inszenium/monitoring-contao-manager-plugin/en.yml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+de:
+    title: Contao Manager Link
+    description: >
+        Add link to Contao Manager fpr Contao Monitoring.
```
Comment (xchs) [inline meta/inszenium/monitoring-contao-manager-plugin/en.yml]: Typo: f**p**r

File: linter/whitelists/default.txt [hunk 2]
```diff
@@ -19,3 +20,4 @@ TinyPNG
 trilobit
 Dalley
 PhotoSwipe
+
```
Comment (Toflar) [inline linter/whitelists/default.txt]: Can you remove this additional line please?

File: meta/inszenium/isotope-export/en.yml [hunk 3]
```diff
@@ -0,0 +1,7 @@
+en:
+    title: Isotope eCommerce Export
+    description: >
+        The extension set new options for export orders.
```
Comment (Toflar) [inline meta/inszenium/isotope-export/en.yml]: What about `This extension provides new options for order exports.`? It doesn't really `set` anything.

File: meta/inszenium/monitoring-cert-plugin/de.yml [hunk 4]
```diff
@@ -0,0 +1,8 @@
+de:
+    title: Monitor Zertifikat
+    description: >
+        Fügt für den Contao Monitoring die Prüfung der Zertifikate hinzu.
```
Comment (Toflar) [inline meta/inszenium/monitoring-cert-plugin/de.yml]: `für den Contao Monitoring`? Wirklich "der" Monitoring?

File: meta/inszenium/monitoring-contao-manager-plugin/de.yml [hunk 5]
```diff
@@ -0,0 +1,6 @@
+de:
+    title: Contao Manager Link
+    description: >
+        Fügt für den Contao Monitoring den Link zum Contao Manager hinzu.
```
Comment (Toflar) [inline meta/inszenium/monitoring-contao-manager-plugin/de.yml]: Auch hier, "der" Monitoring?

**Pull request merged**